airbyte-webapp/src/packages/cloud/locales/en.json

```diff
@@ -120,5 +120,8 @@
 
   "verifyEmail.notification": "You successfully verified your email. Thank you.",
 
+  "trial.remainingTimeMessage": "You are using a trial of Airbyte.  Your trial ends in",
+  "trial.remainingTimeMessage.unit": "days",
```

Comment:
There's a way in react intl to use variables so that these two can be combined into the same string.